docker-compose-local.yaml

```diff
@@ -0,0 +1,39 @@
+version: "3"
```

Comment:
Could name this `docker-compose-infra` because it runs both localstack and the postgres db

Reply:
was following other conversions on the name